### ISSUE TITLE
Block 8 Lecture - Variance

### ISSUE THREAD

**Issue by JacobBumgarner** [BiAPoL/Bio-image_Analysis_with_Python#7]
On slide 15/18 of the descriptive statistics lecture from block 8, I believe that the left side of the equation for the calculation of variance is incorrect. I know this isn't a huge issue, but I wanted to point it out!

<p align='center'>
     <img width="50%" alt="Screen Shot 2022-03-09 at 11 27 02 AM" src="https://user-images.githubusercontent.com/70919881/157485513-ee1f65ab-937c-4719-a1cf-5188cf0c51ff.png">
</p>

On another note, I want to thank the group for making these lectures available. This is a highly valuable resource that I've found real benefit from and have enjoyed using!

**Comment by haesleinhuepf** [comment]
Yes, that makes sense. Thanks for reporting @JacobBumgarner ! We'll fix that during the upcoming semester. 

CC @arpoe
